yjit/Cargo.toml

```diff
@@ -30,6 +30,12 @@ debug = true
 debug-assertions = true
 overflow-checks = true
 
+[profile.stats]
+inherits = "release"
+
+[profile.dev_nodebug]
+inherits = "release"
```

Comment:
I started to feel like I want an option to enable everything of `dev` but `-DRUBY_DEBUG`, which seems similar to the use case of `dev_nodebug`, because `-DRUBY_DEBUG` makes the interpreter a bit too slow.

However, the fact that this inherits `release` and makes the compilation of YJIT itself slower annoys me. WDYT about changing this to `inherits = "dev"`? Or should I keep this and add another one like `dev_noassert`?

Reply:
Generally speaking it's not great to have too many configurations. It becomes hard to test properly when there is conditional compilation everywhere.

Seems you want stats to be enabled, what are the things you would like to have in this profile that are not offered by the stats config ?

Reply:
I'd like to use [`profile.dev`](https://github.com/ruby/ruby/blob/10e4fa3a0f05f72733d132794cedd08906b40196/yjit/Cargo.toml#L27-L31) for possibly faster build, and disasm.

Reply:
It would make sense to have profile.dev_nodebug inherit dev, but set debug=false